Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml

```diff
@@ -6,11 +6,9 @@ configuration:
 - display: Fetch indicators
   name: feed
   required: false
-  defaultvalue: true
```

Comment:
why did you change all default values?

Reply:
dsdk-format command did it. revert this?